examples/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "acuvity-ts-examples",
+  "version": "1.0.0",
+  "main": "basic_text.js",
+  "type": "module",
+  "scripts": {
+    "test": "node basic_file.js && node basic_text.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
as they are already in here, fill them out please. License should be the same as for the main repo (Apache 2.0)